schemas/json/theme.json

```
@@ -889,6 +889,17 @@
 						}
 					},
 					"additionalProperties": false
+				},
+				"filter": {
+					"description": "Filters to attach to the styles",
```

Let's keep this description consistent with the other properties at this level.

```suggestion
					"description": "CSS and SVG filter styles.",
```